src/Controls/tests/Core.UnitTests/KeyboardAcceleratorUnitTests.cs

```diff
@@ -6,19 +6,19 @@
 
 namespace Microsoft.Maui.Controls.Core.UnitTests
 {
-	public class AcceleratorUnitTests : BaseTestFixture
+	public class KeyboardAcceleratorUnitTests : BaseTestFixture
```

Comment:
These tests look like they are for the wrong type? But, we probably need both. The old type is still valid - unless it was added in maui net8 - so we probably want to help people convert from the old Accelerator to the new KeyboardAccelerator.

Reply:
accelerators were only implemented in preview7